haret/tests/utils/arbitrary.rs

```diff
@@ -42,10 +42,10 @@ impl Arbitrary for Op {
         let range = Range::new(0, 100);
         match range.ind_sample(&mut thread_rng()) {
             0...80 => Op::ClientRequest(ClientRequest::arbitrary(g)),
-            80...85 => Op::Commit,
```

Comment:
Ugh, the fact that this is inclusive and not exclusive was something I didn't realize since `..` is exclusive used elsewhere. The difference between 2 and 3 dots could be confusing... Bad rust... bad boy. newspaper on the nose. At least clippy catches this :)